utils.py

```diff
@@ -20,3 +20,22 @@ def log(self, string):
         if self._enabled:
             prefix = "[terminal_view debug] [%.3f] " % (time.time())
             print(prefix + string)
+
+
+class TerminalViewManager():
+    """
+    A manager to control all TerminalView instance.
+    """
+    @classmethod
+    def register(cls, terminal_view):
```

Comment:
Shouldn't this also have an `unregister` class method that's invoked when the terminal_view tab is closed?